### PR TITLE
Cleanup issue template

### PR DESCRIPTION
**Pull request by martypowell**
Based on a sprint planning discussion and talking with @tmccoy529 we have made some changes to the issue template to clarify requirements.

@danfox01 please review this in addition to the development team.

### CODE REVIEW

**Review by tmccoy529 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/request-a-new-component.md [hunk 1]
```diff
@@ -6,7 +6,6 @@ labels: new component
 assignees: ''
 
 ---
-
 **Please name the requested component**
```
Comment (danfox01) [inline .github/ISSUE_TEMPLATE/request-a-new-component.md:9]: `## Name of Component`

File: .github/ISSUE_TEMPLATE/request-a-new-component.md [hunk 2]
```diff
@@ -23,5 +22,23 @@ An example of this would be a `<a href="/services" title="View county services">
 **Example or Mockup**
```
Comment (danfox01) [inline .github/ISSUE_TEMPLATE/request-a-new-component.md:22]: `## Example or Mockup`

File: .github/ISSUE_TEMPLATE/request-a-new-component.md [hunk 3]
```diff
@@ -23,5 +22,23 @@ An example of this would be a `<a href="/services" title="View county services">
...
 Include an example of the component from another site, or a mockup
 
+**Acceptance Criteria**
```
Comment (danfox01) [inline .github/ISSUE_TEMPLATE/request-a-new-component.md:25]: `## Acceptance Criteria`

File: .github/ISSUE_TEMPLATE/request-a-new-component.md [hunk 4]
```diff
@@ -23,5 +22,23 @@ An example of this would be a `<a href="/services" title="View county services">
...
+**Scenario Based**:
+
+**Scenario**: User clears search terms via Escape key
+**Given**: User is actively using search field
+**When**: User clicks Escape Key
+**Then**: Search field values are emptied
+
+...repeat
+
 **Additional context**
```
Comment (danfox01) [inline .github/ISSUE_TEMPLATE/request-a-new-component.md:43]: `## Additional Context`

**Comment by danfox01** [comment]
Late to this party, but the only comment I have is that these might be easier to scan if the major headings are styles as `## Headings` instead of bold text. Especially once you get a lot of rules in there, images, examples of multiple states etc.

**Comment by ascire** [comment]
I'm just a bystander but to add to @danfox01's comment, using headings would allow a document renderer to do fancy automated things like adding a Table of Contents.

**Pull request merged**